### PR TITLE
test: a11y axe wdio poc

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/axe/report.acs-spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+describe('Accessibility test', () => {
+    it('Should check for accessibility issues using Axe', () => {
+        const { title } = new BaseComponentPo();
+        const options = { runOnly: { type: 'tag', values: ['wcag2a', 'wcag2aa'] } };
+        console.log(browser.capabilities.browserVersion);
+        for (const pageUrl in appURLs) {
+            for (const urls in appURLs[pageUrl]) {
+                open(`/fundamental-ngx#/${pageUrl}${appURLs[pageUrl][urls]}`);
+                pause(800);
```
Comment (Seamoo13) [inline e2e/wdio/axe/report.acs-spec.ts:15]: is pause needed? shouldn't waitForPresent(title) be enough?

File: wdio-a11y.conf.js [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+    // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+    // directory is where your package.json resides, so `wdio` will be called from there.
+    //
+    specs: [
+        './e2e/wdio/**/*.acs-spec.ts'
+    ],
+    // Patterns to exclude.
+    exclude: [
+        './e2e/wdio/**/checkbox-group.e2e-spec.ts',
```
Comment (Seamoo13) [inline wdio-a11y.conf.js:27]: do we need to exclude this from accessibility tests?

File: wdio-local.conf.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ exports.config = {
     // directory is where your package.json resides, so `wdio` will be called from there.
     //
     specs: [
-        './e2e/wdio/**/*.e2e-spec.ts'
+        './e2e/wdio/**/*.acs-spec.ts'
```
Comment (Seamoo13) [inline wdio-local.conf.js:24]: maybe leave 'e2e' as the default for the local config and add acs as a commented out option (to help devs)

Comment (AntonOlkhovskyi) [reply wdio-local.conf.js]: Done.

File: wdio-a11y.conf.js (hunk 2 above)
Comment (AntonOlkhovskyi) [reply wdio-a11y.conf.js:29]: Done.

File: e2e/wdio/axe/report.acs-spec.ts (hunk 1 above)
Comment (AntonOlkhovskyi) [reply e2e/wdio/axe/report.acs-spec.ts]: done.

**Review by Seamoo13 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**